srtcore/core.cpp

```diff
@@ -5415,18 +5415,14 @@ int srt::CUDT::rcvDropTooLateUpTo(int seqno)
 {
     // Make sure that it would not drop over m_iRcvCurrSeqNo, which may break senders.
     if (CSeqNo::seqcmp(seqno, CSeqNo::incseq(m_iRcvCurrSeqNo)) > 0)
-        seqno = CSeqNo::incseq(m_iRcvCurrSeqNo);
-
-    const int seq_gap_len = CSeqNo::seqoff(m_iRcvLastSkipAck, seqno);
-
-    // seq_gap_len can be <= 0 if a packet has been dropped by the sender.
-    if (seq_gap_len > 0)
     {
-        // Remove [from,to-inclusive]
-        dropFromLossLists(m_iRcvLastSkipAck, CSeqNo::decseq(seqno));
-        m_iRcvLastSkipAck = seqno;
+        LOGC(tslog.Error, log << CONID() << "IPE: drop request up to %" << seqno << " while last received is %"
```

Comment:
Right. Forgot that this function is also being used from the group synchronization code, and in this case this sequence may exceed the last received sequence. Still, AFAIR this last received sequence has to be synchronized as well. Fortunately this will remain only temporarily, until the common receiver buffer is ready.

Reply:
> this last received sequence has to be synchronized as well

I tried, this may break senders' check: 
```
    if (CSeqNo::seqcmp(ackdata_seqno, CSeqNo::incseq(m_iSndCurrSeqNo)) > 0)
```

Reply:
That's why I'd simply leave it as is for now. The common receiver code will approach this part differently.